src/client/client.ts

```diff
@@ -172,6 +175,27 @@ export class Client extends BaseClient {
     return this.fetchResources("passportScores", {}, passportScoreBuilder);
   }
 
+  getMatchingEstimates(
+    roundId: string,
+    chainId: string,
+    potentialVotes: {
+      contributor: string;
+      recipient: string;
+      amount: bigint;
+    }[],
+  ): Promise<MatchingEstimate[]> {
+    return this.fetchResources(
+      "estimate",
+      { roundId, chainId },
+      matchingEstimateBuilder,
+      // @ts-expect-error Query is required in ReqParams but not actually required in our implementation
```

Comment:
In this case, then, it would seem like the error is in `fetchResources`. Leaky abstraction?

Looking at `fetchResources` in general, it's not clear what's its added value.